src/main/resources/theme/settings/settings.fxml

```diff
@@ -38,7 +39,7 @@
                                 <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES"/>
                                 <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES"/>
                                 <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES"/>
-                                <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES"/>
+                                <RowConstraints minHeight="10.0" vgrow="SOMETIMES"/>
```

Comment:
I don't think this was intentional?

Reply:
it was, before the text only grew to 50 px know this stretches if the text needs to wrap more than 2 lines. I did this there because it can have potentially long texts